app/domain/errors.rb

```diff
@@ -307,8 +307,9 @@ module AuthnAzure
           code: "CONJ00053E"
         )
 
-        MissingProviderFieldsInXmsMirid = ::Util::TrackableErrorClass.new(
-          msg:  "Provider fields are missing in xms_mirid {1}",
+        InvalidProviderFieldsInXmsMirid = ::Util::TrackableErrorClass.new(
+          msg:  "Provider fields are in invalid format in xms_mirid {1}." \
+                "It should have 3 parts",
```

Comment:
Do we need this hint here?

Reply:
I think that without it the message is very ambiguous and doesn't tell the customer much. Without it the customer will think "well, what _is_ a valid format?"